pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
     "pymupdf>=1.25,<2",
     "pyyaml>=6,<7",
     "scipy>=1.7,<2",
-    "svg.path>=4.1,<7",
+    "svg.path>=4.1,<8",
```

Comment:
This will require a news, such that I do not miss this change in the recipe in conda forge.